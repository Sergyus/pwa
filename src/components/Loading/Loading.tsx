import React from 'react';
import styles from './styles.module.scss';

export default function Loading(): JSX.Element {
  return (
    <div className={styles.Loading}>
      <p>Loading...</p>
    </div>
  );
}
