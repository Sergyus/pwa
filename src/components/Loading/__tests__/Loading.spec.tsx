import React from 'react';
import { render, screen } from '@testing-library/react';
import Loading from '..';

describe('<Loading/>', () => {
  it('should renders', () => {
    const { asFragment } = render(<Loading />);
    expect(screen.getByText('Loading...')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
