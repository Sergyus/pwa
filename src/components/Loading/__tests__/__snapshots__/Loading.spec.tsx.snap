// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading/> should renders 1`] = `
<DocumentFragment>
  <div
    class="Loading"
  >
    <p>
      Loading...
    </p>
  </div>
</DocumentFragment>
`;
